src/library/coverart.h

```diff
@@ -51,8 +50,6 @@ class CoverInfo : public CoverInfoRelative {
           trackLocation(tl) {
     }
 
-    virtual ~CoverInfo() {};
```

Comment:
I prefer to define a destructor in derived classes even if it is empty. By declaring it as _override_ instead of _virtual_ the compiler checks that the base class indeed has a virtual destructor.